src/AppInstallerRepositoryCore/Microsoft/Schema/Portable_1_0/PortableTable.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "SQLiteWrapper.h"
+#include "SQLiteStatementBuilder.h"
+#include <string_view>
+
+namespace AppInstaller::Repository::Microsoft::Schema::Portable_V1_0
+{
+    // File type enum of the portable file
+    enum class PortableFileType
+    {
+        Unknown,
+        File,
+        Directory,
+        Symlink
+    };
+
+    // Column type enum of the portable table
+    enum class PortableColumnType
+    {
+        FilePath,
+        FileType,
+        SHA256,
+        SymlinkTarget,
+        IsCreated
+    };
+
+    // Metadata representation of a portable file placed down during installation
+    struct PortableFile

```

Comment:
The schema version erasing type needs a struct like this, which can be used down into the schema specific files.